example/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>Chat Engine Uploadcare Example</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/css/bootstrap.min.css" integrity="sha384-AysaV+vQoT3kOAXZkl02PThvDr8HYKPZhNT5h/CXfBThSRXQ6jW5DO2ekP5ViFdi" crossorigin="anonymous">
+
+    <style type="text/css">
+    * {
+        margin: 0px;
+        padding: 0px;
+    }
+    body {
+        margin: 30px auto;
+    }
+    .log img {
+        max-width: 100%;
+    }
+    form {
+        margin-bottom: 10px;
+    }
+    </style>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <div class="col-sm-6 offset-sm-3">
+            <div class="card">
+                <div class="card-header">
+                    Chat
+                </div>
+                <div class="card-block">
+                    <div class="log" id="output"></div>
+                    <div class="input-group">
+                        <input type="text" class="form-control message" placeholder="Your Message..." id="input" onkeypress="submit(event)">
+                        <span class="input-group-btn"><button class="btn btn-primary" type="submit" onclick="send()">Send</button></span>
+                    </div>
+                    <input type="hidden" role="uploadcare-uploader" data-crop="disabled" data-images-only="true" />
+
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+
+    <script src="./node_modules/chat-engine/dist/latest/chat-engine.js" type="text/javascript"></script>
+    <script src="./node_modules/chat-engine-uploadcare/dist/chat-engine-uploadcare.js" type="text/javascript"></script>
+    <script>
+      UPLOADCARE_LOCALE = "en";
```

Comment:
I think these variables should be in ```script.js``` with a comment explaining what they are and how to get em.